### DISCUSSION TITLE
制卡图片居中

### DISCUSSION THREAD

**Discussion by autokxs** [WatsonYao/nian_app#61] [Q&A] [answered]
制卡功能中有文字居中对齐，能不能也支持图片居中对齐？
另外，图片显示大小是不是可以调整为更合适的大小？目前小小的一个方格，看上去不是很美观

**Comment by WatsonYao** [comment, marked as answer]
感谢反馈，制卡这块没有专业的图文排版软件功能丰富，还在考虑取舍问题。

**Reply by autokxs** [reply]
挺顺手的一个小功能，还是希望可以保留。

**Reply by WatsonYao** [reply]
“挺顺手” 哈哈哈哈哈哈哈
